libs/api/auth/data-access/src/lib/strategies/api-auth-strategy-telegram.module.ts

```diff
@@ -0,0 +1,34 @@
+import { DynamicModule, Logger, Module } from '@nestjs/common'
+import { ApiCoreDataAccessModule } from '@pubkey-link/api-core-data-access'
+import { ApiUserDataAccessModule } from '@pubkey-link/api-user-data-access'
+import { ApiAuthDataAccessModule } from '../api-auth.data-access.module'
+import { ApiAuthStrategyTelegramService } from './api-auth-strategy-telegram.service'
+import { ApiAuthStrategyService } from './api-auth-strategy.service'
+
+@Module({})
+export class ApiAuthStrategyTelegramModule {
+  static logger = new Logger(ApiAuthStrategyTelegramModule.name)
+  static register(): DynamicModule {
```

Comment:
This is kind of a finicky thing to get rid so I didn't bother asking you, what it does is dynamically load the module and log a message when it's not enabled.